### PR TITLE
[incubator/chartmuseum] upgrade to latest version

### PR DESCRIPTION
**Pull request by krancour**
/assign jdolitsky

### CODE REVIEW

File: incubator/chartmuseum/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
 name: chartmuseum
-version: 0.3.6
```
Comment (jdolitsky) [inline incubator/chartmuseum/Chart.yaml:4]: Lets put 0.3.7 here.
Semver in charts is strange to grok, but I think updating Docker image should be a patch bump

**Review by jdolitsky — CHANGES_REQUESTED** [review]
Is it too much to ask to add the new env vars for Alibaba in values.yaml as well?

```
STORAGE_ALIBABA_BUCKET
STORAGE_ALIBABA_PREFIX
STORAGE_ALIBABA_ENDPOINT
STORAGE_ALIBABA_SSE
```

File: incubator/chartmuseum/Chart.yaml (hunk 1 above)
Comment (krancour) [reply incubator/chartmuseum/Chart.yaml:4]: If you're set on that, I'll go with it. I maintain a couple other charts and have had to deal with this a lot. If you don't mind considering an alternative perspective, here it is...

If the Docker image you're bumping contains a new minor revision (i.e. new features), then implicitly, the chart _also_ contains new features-- features which must sometimes be accounted for in the chart, as evidenced by your request for me to include the following in the chart (which I am happy to do, btw):

```
STORAGE_ALIBABA_BUCKET
STORAGE_ALIBABA_PREFIX
STORAGE_ALIBABA_ENDPOINT
STORAGE_ALIBABA_SSE
```

Sleep on it. Ultimately, I'm fine with whatever you decide is right here.

Comment (jdolitsky) [reply incubator/chartmuseum/Chart.yaml:4]: @krancour ok, I think youre right :)

Lets keep with 0.4.0. Will use this reasoning going forward

Comment (krancour) [reply incubator/chartmuseum/Chart.yaml:4]: sgtm. Thanks for listening.

Just added the Alibaba detail as well.

**Pull request merged**